company_website/templates/estimate_project/estimate_project_form.haml

```diff
@@ -4,7 +4,7 @@
   {% include "common_partials/section_header.haml" with title="Estimate your project" subtext="We’ll get to back to you within 24 hours with the next steps" %}
 
 .container.estimate-form
-  %form{:method => "post", :id => "formABC", :onsubmit => 'return gtagReportConversion("{{ URL_PREFIX }}://{{ request.get_host }}{{ thank_you_url }}")'}
+  %form{:method => "post", :id => "formABC"}
```

Comment:
Please change ID of this form to more proper one.

Reply:
Done